protocol-specs/generator_outputs/lisk_codec/validArrayEncodings.json

```diff
@@ -0,0 +1,122 @@
+{
+	"title": "Valid array encodings",
+	"summary": "Examples of encoding arrays as required by lisk-codec",
+	"config": {
+		"network": "devnet"
+	},
+	"runner": "lisk_codec",
+	"handler": "validArrayEncodings",
+	"testCases": [
+		{
+			"description": "Encoding of boolean types",
+			"config": {
+				"network": "devnet"
+			},
+			"input": {
+				"arrayInts": {
+					"object": {
+						"list": [
+							3,
+							1,
+							4,
+							1,
+							5,
+							9,
+							2,
+							6,
+							5
+						]
+					},
+					"schema": {
+						"type": "object",
+						"properties": {
+							"list": {
+								"dataType": "array",
+								"fieldNumber": 1
+							}
+						}
+					}
+				},
+				"arrayBools": {
+					"object": {
+						"list": [
+							true,
+							true,
+							false,
+							true,
+							false,
+							false
+						]
+					},
+					"schema": {
+						"type": "object",
+						"properties": {
+							"list": {
+								"dataType": "array",
```

Comment:
```suggestion
								"type": "array",
```

Reply:
Had to make the fix in the generator itself.